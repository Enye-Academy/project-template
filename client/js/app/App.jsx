import React from 'react';
import { Router } from 'react-router';
import { Route, Switch } from 'react-router-dom';
import history from '../utils/history';

const App = () => (
  <Router history={history}>
    <Switch>
      <Route
        exact
        path="/"
        name="landing"
        component={(Home)}
      />
    </Switch>
  </Router>
);
export default App;
