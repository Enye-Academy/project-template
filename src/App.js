import React from 'react';

function App() {
    return <h1>WELCOME TO VOTE RIGHT APPLICATION</h1>;
}

export default App;
