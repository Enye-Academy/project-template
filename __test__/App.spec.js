import React from 'react';
import toJson from 'enzyme-to-json';
import App from '../src/App';
import { mount } from './enzyme';

describe('SAMPLE TEST', () => {
    it('should return a successful sample test ', () => {
        expect(true).toBeTruthy();
    });

    it('should render the App component correctly', () => {
        const Wrapper = mount(<App />);
        expect(toJson(Wrapper)).toMatchSnapshot();
        expect(Wrapper.find('h1')).toHaveLength(1);
    });
});
