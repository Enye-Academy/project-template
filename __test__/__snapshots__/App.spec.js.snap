// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SAMPLE TEST should render the App component correctly 1`] = `
<App>
  <h1>
    WELCOME TO VOTE RIGHT APPLICATION
  </h1>
</App>
`;
